src/JuMPModels/quadrotorP2P.jl

```diff
@@ -0,0 +1,98 @@
+"""
+Quadrotor Problem:
+    We want to find the optimal trajectory of a quadrotor to reach a target position.
+    The objective is to minimize the final time.
+    The problem is formulated as a JuMP model, and can be found [here](https://arxiv.org/pdf/2303.16746)
+"""
+function quadrotorP2P(;nh::Int64=100)
```

Comment:
I would suggest finding a better name for the function. Nobody knows what `P2P` means here